testimonials/index.php

```diff
@@ -1,29 +1,29 @@
 <?php
   require_once('includes/template.php');
-  
+
   // Required
   head([
     'title' =>'Testimonials | Keyman',
     'css' => ['template.css'],
     'showMenu' => true
-  ]);           
+  ]);
 ?>
 <h2 class="red underline">What people are saying about Keyman Desktop</h2>
 
 <p class="center">
   <span class="testimonial">"... I would suggest to stick with keyman and I am sure you would love the experience.<br/>
   I type #Amharic like a Pro on Keyman [winking face]"</span>
-  <h4 class="caption"><a href="https://twitter.com/MekonnenAkele/status/1208281934721409025">@MekonnenAkele</a> - December 2019</h4>
+  <h4 class="caption"><a href="https://twitter.com/MekonnenAkele/status/1208281934721409025" target="_blank">@MekonnenAkele</a> - December 2019</h4>
```

Comment:
I thought previously you wanted to get away from having `target="_blank"` in the links.

Or is it to use for external links?

Reply:
For these external links, I am okay with `_blank`. Most places, though, nope :)